src/hello_client.erl

```diff
@@ -165,12 +166,16 @@ handle_call(terminate, _From, State) ->
 %% @hidden
 handle_info({?INCOMING_MSG, Message}, State) ->
     incoming_message(Message, State);
+handle_info(?PING, State = #client_state{waiting_for_pong = true, keep_alive_interval = KeepAliveInterval}) ->
+    ?LOG_ERROR("There is no PONG answer on PING for ~p msec. Client will be restarted.", [KeepAliveInterval]),
```

Comment:
@skattelmann 
1) Transport specific thing will happen in transport specific modules. When we are stopping client with error reason we call transport_terminate/2 and in zmq case it will close socket.
2) Do you mean that we need to save timestamp between client restarts just for logging it?  

Maybe we should think about some metrics for avg/min/max time for ping-pong?